src/Features/VisualBasic/Portable/MakeMethodSynchronous/RemoveAsyncModifierHelpers.vb

```diff
@@ -0,0 +1,55 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
```

Comment:
is github not classifying this file for you either?

Reply:
Seems okay here
![image](https://user-images.githubusercontent.com/754264/87400165-ce7d4200-c5fb-11ea-8862-328a7d40d14f.png)